libraries/ms-drivers/test/test_pca9539r_gpio_expander.c

```diff
@@ -308,3 +310,17 @@ void test_pca9539r_gpio_get_state_invalid_address(void) {
   Pca9539rGpioState state;
   TEST_ASSERT_EQUAL(STATUS_CODE_INVALID_ARGS, pca9539r_gpio_get_state(&address, &state));
 }
+void prv_test(const struct GpioAddress *address, void *context) {
+  GpioState state = GPIO_STATE_LOW;
+  gpio_get_state(address, &state);
+}
```

Comment:
A few points about this function:
* The name `prv_test` doesn't really describe what its purpose is, which is to be a callback for subscribing to interrupts. Please change its name to reflect its purpose.
* It's a private function, so it should be `static`.
* You don't need the `struct` keyword before `GpioAddress`; we do the `typedef struct GpioAddress { ... } GpioAddress` construct so you don't have to say `struct` everywhere.
* The contents of the function aren't really doing anything useful; see my previous comment for what to replace them with.